src/cmd/policy.go

```diff
@@ -133,6 +139,37 @@ func RegoFuncGetGithubRepo(ctx rego.BuiltinContext, a, b *ast.Term) (*ast.Term,
 	return ast.NewTerm(v), nil
 }
 
+func RegoFuncGetMaturity(ctx rego.BuiltinContext, a, b *ast.Term) (*ast.Term, error) {
+
+	var alias, level string
+	if err := ast.As(a.Value, &alias); err != nil {
+		return nil, err
+	}
+	if err := ast.As(b.Value, &level); err != nil {
+		return nil, err
+	}
+
+	if alias == "" {
+		log.Error().Msgf("opslevel.service_maturity_is(\"%s\", \"%s\") failed: Please provide a valid alias", alias, level)
+		return nil, nil
+	}
+
+	if level == "" {
+		log.Error().Msgf("opslevel.service_maturity_is(\"%s\", \"%s\") failed: Please provide a valid repo", alias, level)
+		return nil, nil
+	}
+
+	client := getClientGQL()
+	service, err := client.GetServiceMaturityWithAlias(alias)
+	cobra.CheckErr(err)
+	serviceLevel := service.MaturityReport.OverallLevel.Alias
+	if serviceLevel == level {
```

Comment:
So we have to do a more complex check here.  The way i see this being used is to ensure the service level is `>= "silver"`. Maybe that means we also need to change the name of the function too? 🤷 

So we may need to use the level index or something and do a >= test.

Reply:
Ah ok, that's a good point.  Would it be better to change it to just return the level and let the policy writer add whatever logic they want in their rego policy to gate deployments?

Reply:
@eochoa260 ouu thats a great idea - that means is more like "get service level index" so they can choose what kind of logic to run on it `>=` or `<=` or even just `==`. ❤️

Reply:
Ok, updated it to return the OverallLevel for a service as an object so users can get the Name, Index, Alias, etc and run any comparison they want in the Rego policy. 😃